### PR TITLE
Create robots page

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/manage/fleet/robots.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Builder provides a more graphical UI.
+Raw JSON mode can be useful for users who are familiar with writing JSON, and for seeing and editing everything at once.
+
+Within the **CONFIG** tab (in builder mode) are separate sub-tabs for components, services, remotes, processes, network, authentication, and fragments, so be sure to configure these in their respective tabs.
+In raw JSON mode, all of these resources are displayed in a single text field.
+
+Details on configuring specific hardware components and services can be found in these topics:
+
+- [Components](/components/)
+- [Services](/services/)
```
Comment (JessamyT) [inline docs/manage/fleet/robots.md:67]:
```suggestion
Here you can start adding robot components, services, and other robot resources.

Find more information [in the configuration documentation](../../manage/configuration/#the-config-tab).
```

**Review by JessamyT — COMMENTED** [review]
Not reviewing all the content but including the one suggestion since that page just landed. Also, consider changing the weight of locations. Not sure if you want robots-locations-organizations or the other way around.

File: docs/manage/fleet/robots.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+You can delete a robot by checking the **Sure?** box in the lower left of the robot page and clicking **DELETE ROBOT**.
+
+![The DELETE ROBOT button and the confirmation checkbox (Sure?) next to it.](../../img/app-usage/delete.png)
+
+## Navigating the robot page
+
+The banner at the top of the robot page displays the robot's location, name, and a drop down list of all parts of that robot.
+The first part you create will be the _main part_ but you can create additional parts in the drop down.
+
+{{< figure src="../../../img/app-usage/part-drop-down.png" width="800px" alt="The robot page for an example robot with the parts drop down open." title="Example Robot Page with the Parts drop-down open (boxed in red)." >}}
```
Comment (npentrel) [inline docs/manage/fleet/robots.md:34]:
```suggestion
{{< figure src="../../img/app-usage/part-drop-down.png" width="800px" alt="The robot page for an example robot with the parts drop down open." title="Example Robot Page with the Parts drop-down open (boxed in red)." >}}
```

File: docs/manage/fleet/robots.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+To delete a part or make it the main part, use the buttons in the top right of the **CONFIG** tab.
+
+{{< figure src="../../../img/app-usage/part-mgmt.png" width="800px" alt="Screenshot of the CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons." title="Screenshot of the CONFIG tab of a robot's page highlighting the Make main part and Delete Part buttons (boxed in red)." >}}
```
Comment (npentrel) [inline docs/manage/fleet/robots.md:38]:
```suggestion
{{< figure src="../../img/app-usage/part-mgmt.png" width="800px" alt="Screenshot of the CONFIG tab of a robot's page noting the location of the Make main part and Delete Part buttons." title="Screenshot of the CONFIG tab of a robot's page highlighting the Make main part and Delete Part buttons (boxed in red)." >}}
```

File: docs/manage/fleet/robots.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+This is done through WebRTC.
+For example, if you have configured a base with wheels, you can control your robot's movement with an arrow pad and fields to change base’s speed.
+If you have configured a camera component, a window in the **CONTROL** tab displays the camera output.
+
+### Security
+
+The **SECURITY** tab allows you to access and exchange the **Robot Part Secret Keys** of your robot.
+A robot part secret is a unique secret used by the robot to communicate with the cloud.
+
+{{< figure src="../../../img/app-usage/robot-part-secret-keys-drop-down.png" width="800px" alt="Screenshot of the SECURITY tab of a robot's page noting the Robot Part Secret Keys drop-down menu, with the clipboard icon on the far right and the Generate Key button underneath the drop-down." title="Screenshot of the SECURITY tab of a robot's page." >}}
```
Comment (npentrel) [inline docs/manage/fleet/robots.md:96]:
```suggestion
{{< figure src="../../img/app-usage/robot-part-secret-keys-drop-down.png" width="800px" alt="Screenshot of the SECURITY tab of a robot's page noting the Robot Part Secret Keys drop-down menu, with the clipboard icon on the far right and the Generate Key button underneath the drop-down." title="Screenshot of the SECURITY tab of a robot's page." >}}
```

**Review by JessamyT — APPROVED** [review]
Looks good unless do you want to add a box to the manage index page for config a robot?

**Comment by npentrel** [comment]
Oh - yes absolutely. Let me do that in a new PR and also look whether I can briefly fix the robots page up further

**Pull request merged**